rosbag2_transport/include/rosbag2_transport/player.hpp

```diff
@@ -87,11 +99,11 @@ class Player : public rclcpp::Node
   // Playback control interface
   /// Pause the flow of time for playback.
   ROSBAG2_TRANSPORT_PUBLIC
-  void pause();
+  virtual void pause();
```

Comment:
After some consideration I have to step back in my request to avoid virtual methods. It seems there are no good way to mock those methods without defining them as virtual. 
Let's keep them virtual.